src/java_bytecode/java_qualifiers.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 DiffBlue Limited. All Rights Reserved.
+
+/// \file
+/// Java-specific type qualifiers
+
+#ifndef CPROVER_JAVA_BYTECODE_JAVA_QUALIFIERS_H
+#define CPROVER_JAVA_BYTECODE_JAVA_QUALIFIERS_H
+
+#include "java_types.h"
+#include <ansi-c/c_qualifiers.h>
+
+class java_qualifierst : public c_qualifierst
+{
+public:
+  explicit java_qualifierst(const namespacet &ns);
+
+  java_qualifierst &operator=(const java_qualifierst &other);
+  java_qualifierst &operator=(java_qualifierst &&other);
+  virtual c_qualifierst &operator=(const c_qualifierst &other) override;
```

Comment:
`virtual` redundant with `override`

Reply:
This seems like good redundancy from a documentation perspective. Being able to see whether a function is virtual in a single place without having to check the beginning and the end of the declaration is a benefit in my view. As long as it's consistent within a class I think there are examples of both using and not using `virtual` with `override` in CBMC. Do you want to make a general policy on this or accept that it's a style issue on which developers can disagree?

Reply:
If there is a decision to avoid `virtual ... override` then a7c24fb63 should be revisited: `cpplint.py` would happily warn about those, but this had been commented out (by @peterschrammel  :-) ). Actually there is more to revisit when it comes to a7c24fb63, such as operator spacing...